include/wd_aead.h

```diff
@@ -0,0 +1,147 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+#ifndef __WD_AEAD_H
+#define __WD_AEAD_H
+
+#include <dlfcn.h>
+#include "wd_alg_common.h"
+#include "config.h"
+#include "wd_cipher.h"
+#include "wd_digest.h"
+#include "wd.h"
+
+/**
+ * wd_aead_op_type - Algorithm type of option
+ */
+enum wd_aead_op_type {
+	WD_CIPHER_ENCRYPTION_DIGEST,
+	WD_CIPHER_DECRYPTION_DIGEST,
+	WD_DIGEST_CIPHER_ENCRYPTION,
+	WD_DIGEST_CIPHER_DECRYPTION,
+};
+
+/**
+ * wd_cipher_type - Algorithm type of cipher
+ * statement in wd_cipher.h
+ *
+ * wd_cipher_mode - Algorithm mode of cipher
+ * statement in wd_cipher.h
+ */
+
+struct wd_aead_sess_setup {
```

Comment:
add descriptions

Reply:
You only need to add a comment when the meaning is not clear. 
The meaning here is very straightforward and no comment is required